web/app/components/document/sidebar.hbs

```diff
@@ -313,64 +313,8 @@
       </div>
     </div>
 
-    {{!-- <div class="flex flex-col items-start space-y-2 px-3">
-      <small class="hds-typography-body-100 hds-foreground-faint">Tags</small>
-      {{#if this.isOwner}}
-        <EditableField
-          @value={{this.tags}}
-          @onChange={{perform this.save "tags"}}
-          @loading={{this.save.isRunning}}
-        >
-          <:default>
-            {{#if this.tags.length}}
-              <ol>
-                {{#each this.tags as |tag|}}
-                  <li>
-                    <FlightIcon
-                      @name="tag"
-                      @color="var(--token-color-palette-neutral-500)"
-                    />
-                    {{tag}}
-                  </li>
-                {{/each}}
-              </ol>
-            {{else}}
-              <em>No tags</em>
-            {{/if}}
-          </:default>
-          <:editing as |F|>
-            <Inputs::TagSelect
-              class="multiselect--narrow"
-              @selected={{this.tags}}
-              @onChange={{this.updateTags}}
-              {{click-outside (fn F.update this.tags)}}
-            />
-          </:editing>
-        </EditableField>
-      {{else}}
-        {{#if this.tags.length}}
-          <ol>
-            {{#each this.tags as |tag|}}
-              <li>
-                <FlightIcon
-                  @name="tag"
-                  @color="var(--token-color-palette-neutral-500)"
-                />
-                {{tag}}
-              </li>
-            {{/each}}
-          </ol>
-        {{else}}
-          <em>No tags</em>
-        {{/if}}
-      {{/if}}
-    </div> --}}
-
```

Comment:
It's time. (Nothing here we can't easily replicate later if we want to implement tags.)